### PR TITLE
Use the engine url for the api

### PR DESCRIPTION
**Pull request by shem8**
Use the engine url param to call the right backend

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -3566,10 +3566,17 @@ const core = __nccwpck_require__(186);
 const exec = __nccwpck_require__(514);
 const { promises: fs } = __nccwpck_require__(747);
 
-const host =
-  core.getInput("owner") === "Staging-ObelusFamily"
-    ? "https://engine-staging.wilco.gg"
-    : "https://engine.wilco.gg";
```
Comment (stellaanavi) [inline dist/index.js:3572]: Why dist is part of the PR?

Comment (shem8) [reply dist/index.js:3572]: it automatically compile the code so we can test it

**Review by esapir — APPROVED** [review]

File: dist/index.js (hunk 1 above)
Comment (liorbrauer) [reply dist/index.js:3572]: why do we have dist in the repo? 🤔 should be added to `.gitignore`, no?

Comment (liorbrauer) [reply dist/index.js:3572]: ah oops this isn't wilco-engine

File: index.js [hunk 2]
```diff
@@ -2,10 +2,17 @@ const core = require("@actions/core");
...
-  core.getInput("owner") === "Staging-ObelusFamily"
-    ? "https://engine-staging.wilco.gg"
-    : "https://engine.wilco.gg";
+const host = () => {
+   if (core.getInput("owner") === "Staging-ObelusFamily") {
+     const company = core.getInput("repo").split("-")[0]; 
+     if (company === "Anythink") {
+       return "https://engine-staging.wilco.gg";
+     } else {
+       return `https://${company}.ngrok.io`;
```
Comment (liorbrauer) [inline index.js:11]: what does `company` equal to locally? I'm confused. Could you describe in the PR description how it's used? do I have to start ngrok with a custom domain based on my repo name?

File: index.js [hunk 3]
```diff
@@ -2,10 +2,8 @@ const core = require("@actions/core");
 const exec = require("@actions/exec");
 const { promises: fs } = require("fs");
 
-const host =
-  core.getInput("owner") === "Staging-ObelusFamily"
-    ? "https://engine-staging.wilco.gg"
-    : "https://engine.wilco.gg";
+const host = 
+  core.getInput("engine") || (core.getInput("owner") === "Staging-ObelusFamily" ? "https://engine-staging.wilco.gg" : "https://engine.wilco.gg");
```
Comment (darmalovan) [inline index.js:6]: What's the value of `core.getInput("engine")` It's the env var?

Comment (shem8) [reply index.js:6]: yes, see here: https://github.com/trywilco/wilco-engine/pull/1553

**Review by darmalovan — APPROVED** [review]

**Pull request merged**